README.md

```diff
@@ -358,6 +471,8 @@ The source file is `sources/state2.ml`.
 
 ## 3. Delimited Continuations: A deep dive
 
+**EDITOR'S NOTE: The implementation has changed since this section was written. Results in gdb will differ, but the concepts of the implementation remain mostly the same.**
```

Comment:
This is expected. Does it make sense to update the gdb section with current stack traces?

Reply:
It makes sense to update the gdb section with the current stack traces.

Reply:
I don't know much about gdb or the low-level implementation of OCaml, so I would need help understanding the current implementation and writing the section.

Reply:
done in https://github.com/ocamllabs/ocaml-effects-tutorial/pull/7/commits/6249d7596b530a4da2f926c4c5d96539e19a6254